config/plugins/tours/core.galaxy_ui.yaml

```diff
@@ -51,20 +51,25 @@ steps:
     - title: "Tools"
       component: _.left_panel
       intro: "This is your ToolBox. All tools available in your Galaxy instance you can find here."
+      # Ensure tool activity panel is open
+      prerequisite: "#activity-tools"
 
     - title: "Tool categories"
       component: tools.title
       intro: "Tools are grouped into categories to make them easier to find."
+      prerequisite: "#activity-tools"
 
     - title: "Tool search bar"
       component: tools.search
       intro: "You can search for tools by keywords."
       textinsert: 'filter'
+      prerequisite: "#activity-tools"
```

Comment:
Is there anyway I can convince you these should be components instead of raw CSS selectors (https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tours/_impl.py#L47) or to implement these prerequisites in the testing framework (https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/selenium/navigates_galaxy.py#L2478).

Reply:
Absolutely, just wanted to tease the idea of these "prerequisites" first. 

We should definitely used components instead